app/style/content/conversation/conversation-input.less

```diff
@@ -167,3 +167,27 @@
 .conversation-input-paste-cancel {
   margin-right: 32px;
 }
+
+.conversation-input-emoji-list {
+  z-index: @z-index-level-7;
+  position: fixed;
+  display: block;
+  box-shadow: 0px 2px 10px rgba(0, 0, 0, 0.2);
```

Comment:
even though the designers will have the final say here.. at least style wise we use this:`rgba(0,0,0,.2)`